themes/bakke/README.md

```diff
@@ -0,0 +1,15 @@
+# Bakke-Dark
+
+![alt text](bakke-dark.png)
+
+- **Version Control State Indicators:** It uses brown to indicate a dirty state and bold green for a clean state in version control-related prompts such as Git.
+
```

Comment:
Can you remove those empty lines between the list items? In READMEs of the other themes, they are not present.